SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMapGeneration.cpp

```diff
@@ -52,22 +54,36 @@ bool GenerateTestMaps()
 	TArray<UClass*> TestMapClasses;
 	for (TObjectIterator<UClass> Iter; Iter; ++Iter)
 	{
-		if (Iter->IsChildOf(UGeneratedTestMap::StaticClass()) && *Iter != UGeneratedTestMap::StaticClass())
+		if (Iter->IsChildOf(UGeneratedTestMap::StaticClass()) || Iter->IsChildOf(ASpatialFunctionalTest::StaticClass()))
```

Comment:
You lost the second if condition or the original.

Reply:
Still early days in understanding this PR - aka it seems strange to me to be conflating a check for 'GeneratedTestMap' with 'SpatialFunctionalTest'?